packages/extension/src/utils/fetchPSInfo.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum PSInfoKey {
+  'PrivateStateToken' = 'private-state-token',
+  'Topics' = 'topics',
+  'ProtectedAudience' = 'protected-audience',
```

Comment:
Is it being used?

Reply:
Yes, these are keys used as props for the infoCard component, used [here](https://github.com/GoogleChromeLabs/ps-analysis-tool/blob/efaf423149d78a1646f7e4a089f43bb05fd06be5/packages/extension/src/view/devtools/components/tabContent/topics/index.tsx#L30)

Not all the keys are used right now, but these are kept for inclusion in the future as a prop if required. Plus data object related to each key is present in the PSInfo JSON file.